### PR TITLE
Add mali-bifrost-g31-libgl-gbm

### PR DESCRIPTION
**Pull request by valadaa48**

### CODE REVIEW

**Comment by kmihelich** [comment]
What is this used with? What you have you tested it with?

This is also missing a buildarch setting to restrict to AArch64 builds.

File: alarm/mali-bifrost-g31-libgl/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Maintainer: valadaa48 <valadaa48@gmx.com>
+
+_commit=8867101e509ec9e99c2330b1dbd9830ec9350ffe
+_pkgname=libmali
+pkgbase=mali-bifrost-g31-libgl
```
Comment (kmihelich) [inline alarm/mali-bifrost-g31-libgl/PKGBUILD:5]: pkgbase is not necessary with packages that aren't split.

**Comment by valadaa48** [comment]
I removed the use of pkgbase and added the missing buildarch.

File: alarm/mali-bifrost-g31-libgl/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Maintainer: valadaa48 <valadaa48@gmx.com>
+
+_commit=8867101e509ec9e99c2330b1dbd9830ec9350ffe
+_pkgname=libmali
+pkgname="mali-bifrost-g31-libgl-gbm"
+pkgver=r6p0.01rel0
+pkgrel=1
+pkgdesc="Mali GPU library used in Rockchip Platform"
+arch=('aarch64')
+buildarch=8
```
Comment (kmihelich) [inline alarm/mali-bifrost-g31-libgl/PKGBUILD:10]: Move this up to the top so it's obvious.  See usage in other PKGBUILDs.

**Comment by kmihelich** [comment]
Base directory should match the pkgname.  The -gbm suffix seems unnecessary.

**Comment by valadaa48** [comment]
Good catch, thanks!

**Pull request merged**